src/app/components/search/search.component.ts

```diff
@@ -41,25 +41,26 @@ const FieldErrors: { [key: string]: { [key: string]: string } } = {
   templateUrl: './search.component.html',
   styleUrls: ['./search.component.scss'],
   standalone: true,
-  imports: [CommonModule, ReactiveFormsModule, ResultsComponent, CourthouseComponent],
+  imports: [CommonModule, ReactiveFormsModule, ResultsComponent, CourthouseComponent, SearchErrorComponent],
 })
 export class SearchComponent implements OnInit, AfterViewChecked, OnDestroy {
   @ViewChild(CourthouseComponent) courthouseComponent!: CourthouseComponent;
 
   dateInputType: 'specific' | 'range' | undefined;
-  cases: Case[] = [];
-  loaded = false;
   isSubmitted = false;
   errorSummary: ErrorSummaryEntry[] = [];
-  errorType = '';
-  error = '';
   isAdvancedSearch = false;
   datePatternValidator = Validators.pattern(/^(0[1-9]|[12][0-9]|3[01])\/(0[1-9]|1[0-2])\/\d{4}$/);
   dateValidators = [this.datePatternValidator, futureDateValidator];
   courthouses$ = this.caseService.getCourthouses();
+  courthouse = '';
+
+  // Retrieve Previous Search Results
+  searchResults$ = this.caseService.searchResults$;
+  searchError$: Observable<string> | null = null;
   subs: Subscription[] = [];
 
-  constructor(private caseService: CaseService) {}
+  constructor(public caseService: CaseService) {}
```

Comment:
Why is this public now?

Reply:
It shouldn't be, I set it to public to access it from template but then didn't need to. I will revert.